reactrestaurant/src/components/carousel.js

```diff
@@ -1,44 +1,44 @@
 import React from 'react';
 
-function Carousel() {
+function Carousel(props) {
     return (
-        <div class="bd-example">
-            <div id="carouselExampleCaptions" class="carousel slide" data-ride="carousel">
-                <ol class="carousel-indicators">
-                    <li data-target="#carouselExampleCaptions" data-slide-to="0" class="active"></li>
+        <div className="bd-example">
```

Comment:
Let's also consider how to break these carousel pieces into child components. In this example, we have a fixed number, but with a little bit of tweaking we could be able to handle n carousel items no problem